sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -3054,11 +3054,11 @@
         "properties": {
           "status": {
             "format": "int32",
-            "description": "Method invocation result status.",
+            "description": "Method invocation result status, provided by the device.",
```

Comment:
Why do we need to say provided by the device? Is the method not invoked on the device?

Reply:
It is to highlight that this is the status returned by the device, and not the http status.

Reply:
To clarify this point, there are two status codes that a user can see when making a device method invocation. The first is the http level status code that service might give if the device is not online, the second is the application level status code that only the device can provide

Reply:
What do these status look like? Can you give some examples for me to understand?

Reply:
This is a sample response:
```csharp
public const string DeviceResponseJson = "{\"name\":\"e2e_test\"}";
new MethodResponse(Encoding.UTF8.GetBytes(DeviceResponseJson), 200)
```
The status here is `200`, but it can be any integer as defined by the application's logic.